pkg/proxy/service.go

```diff
@@ -119,6 +121,16 @@ func (info *BaseServiceInfo) OnlyNodeLocalEndpoints() bool {
 	return info.onlyNodeLocalEndpoints
 }
 
+// OnlyNodeLocalEndpointsForInternal is part of ServicePort interface
+func (info *BaseServiceInfo) OnlyNodeLocalEndpointsForInternal() bool {
```

Comment:
We could also rename these accessor methods like `NodeLocalInternal` and `NodeLocalExternal`

Reply:
done